src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+    </div>
```

Comment:
The indentation within your HTML elements seems inconsistent. According to the checklist, you should use 2 spaces for indentation and maintain correct indentation between parent and child elements.